src/odla_layer.h

```diff
@@ -7,7 +7,7 @@
 typedef layer odla_layer;
 
 odla_layer make_odla_layer(int batch, int w, int h, int c,
-                            odla_params params);
+                            odla_params params, network *net);
```

Comment:
where are we using "net" in make_odla_layer?

Reply:
Net is used to get layer pointer, from_odla_tensor output tensor and its related information.